spec/models/user_spec.rb

```diff
@@ -2188,6 +2188,18 @@ def hash(password, salt)
         expect(message.data[:all_unread_notifications_count]).to eq(2)
         expect(message.data[:grouped_unread_notifications]).to eq({ 1 => 1, 15 => 1 })
       end
+
+      it "adds new_personal_messages_notifications_count to the payload" do
+        Fabricate(:notification, user: user, notification_type: Notification.types[:private_message], read: false)
+
+        messages = MessageBus.track_publish("/notification/#{user.id}") do
+          user.publish_notifications_state
+        end
+        expect(messages.size).to eq(1)
+
+        message = messages.first
+        expect(message.data[:new_personal_messages_notifications_count]).to eq(1)
+      end
```

Comment:
I don't think we need another test case here and can just combine this with the test above which we did previously as well.